pandas/core/arrays/base.py

```diff
@@ -813,7 +813,12 @@ def unique(self: ExtensionArrayT) -> ExtensionArrayT:
         uniques = unique(self.astype(object))
         return self._from_sequence(uniques, dtype=self.dtype)
 
-    def searchsorted(self, value, side="left", sorter=None):
+    def searchsorted(
+        self,
+        value: ArrayLike,
```

Comment:
i think scalars are accepted?

Reply:
You're right.  Not documented that way in numpy, so updated in next commit.